src/pages/SearchPage.js

```diff
@@ -70,6 +73,13 @@ class SearchPage extends Component {
         };
     }
 
+    onClose() {
+        if (['web', 'desktop'].includes(getPlatform())) {
```

Comment:
This pattern should be avoided. The proper pattern to follow can be found in our readme: https://github.com/Expensify/Expensify.cash#platform-specific-file-extensions

- Is there some reason that this should be scoped to only the web and desktop platforms?
- Maybe this logic is better to put inside of `modalHide()` if it is  indeed necessary?